pybrightness/module.py

```diff
@@ -14,7 +14,7 @@ class Settings:
 
     """
 
-    operating_system: str = platform.system()
+    operating_system = str(platform.system())
```

Comment:
If you look at the [docs](https://docs.python.org/3/library/platform.html#platform.system) `platform.system()` always returns a string if the OS is identifiable and an empty string otherwise, so there is no need to quote it into a string once again. The `: str` is only a type hint so my IDE knows the attributes the variable `operating_system` can hold.